book/06-github/sections/3-maintaining.asc

```diff
@@ -108,26 +108,26 @@ image::images/maint-03-email-resp.png[電郵回覆]
 如果這只是個簡單的合併，你可以按下 GitHub 網站上的「Merge」按鈕來解決。
 這會做一個「非快進」的合併——即便是可以快進，仍會建立合併提交。
 這意味著，不論你在何種情況按下「Merge」，都會建立一個合併提交。
-如果你點了提示連結，GitHub 就會會提供你所有的資訊，就像你在 <<_merge_button>> 看到的一樣。
+如果你點了提示連結，GitHub 就會會提供你所有的資訊，就像你在 <<r_merge_button>> 看到的一樣。
 
-[[_merge_button]]
+[[r_merge_button]]
 .Merge 按鈕和手動合併 Pull Request 的指引。
 image::images/maint-02-merge.png[Merge 按鈕]
 
 如果你決定不要合併這個 Pull Request，你只需要關閉這個 Pull Request 即可，同時建立這個 Pull Request 的人也會收到通知。
 
-[[_pr_refs]]
+[[r_pr_refs]]
 ===== Pull Request 參照
 
 如果你要處理 *非常多* 的 Pull Request 而且不想加入一堆的遠端或是一直做只會用到一次的拉取，關於這點 GitHub 提供了一個好用的小技巧給你用。
-這是個有點進階的技巧，所以我們會在 <<_refspec>> 提到更多的細節，不過還是非常的有用。
+這是個有點進階的技巧，所以我們會在 <<ch10-git-internals#r_refspec>> 提到更多的細節，不過還是非常的有用。
 
 事實上 GitHub 會把倉儲的 Pull Request 當成伺服器上的假分支。
 預設情況下你不會在拓製的時候取得它們，但他們還是以隱藏的狀態存在著而且你可以用非常簡單的方式取得它們。
 
-To demonstrate this, we're going to use a low-level command (often referred to as a ``plumbing'' command, which we'll read about more in <<_plumbing_porcelain>>) called `ls-remote`.
+To demonstrate this, we're going to use a low-level command (often referred to as a ``plumbing'' command, which we'll read about more in <<ch10-git-internals#r_plumbing_porcelain>>) called `ls-remote`.
 This command is generally not used in day-to-day Git operations but it's useful to show us what references are present on the server.
-為了展示這個事實，我們要使用一個比較低階的指令（通常會被稱作「底層」指令，關於這點我們會在 <<_plumbing_procelain>> 再做詳細描述）—— `ls-remote`。
+為了展示這個事實，我們要使用一個比較低階的指令（通常會被稱作「底層」指令，關於這點我們會在  再做詳細描述）—— `ls-remote`。
```

Comment:
should delete those 2 spaces?

Reply:
No there was a typo: `procelain` instead of `porcelain`.